tests/CMakeLists.txt

```diff
@@ -2,5 +2,5 @@ set(test_srcs test_w3emc.pf)
 
 add_pfunit_ctest (w3emc_test
   TEST_SOURCES test_w3emc_mod.pf
-  LINK_LIBRARIES w3emc_d NetCDF::NetCDF_Fortran sigio_4 nemsio
+  LINK_LIBRARIES w3emc::w3emc_d NetCDF::NetCDF_Fortran sigio::sigio nemsio::nemsio
```

Comment:
shouldn't just `w3emc::w3emc_d` be sufficient, since the rest are dependencies within `w3emc`

Reply:
Yes